src/olcs/contrib/manager.js

```diff
@@ -245,6 +245,41 @@ olcs.contrib.Manager = class extends ol.Observable {
   }
 
 
+  /**
+   * Enable ol3d with a view built from parameters.
+   *
+   * @export
+   * @param {number} lon
+   * @param {number} lat
+   * @param {number} elevation
+   * @param {number} headingDeg
+   * @param {number} pitchDeg
+   * @returns {Promise<undefined>}
+   */
+  set3dWithView(lon, lat, elevation, headingDeg, pitchDeg) {
```

Comment:
To make it in cleaner, let's pass destination and orientation as parameters here.

Reply:
You are right Cesium is not loaded so disregard my suggestion.